tests/foreman/api/test_host.py

```diff
@@ -70,27 +63,29 @@ def test_create_owner_type(self, owner_type):
         correct.
 
         """
-        if owner_type == 'Usergroup' and bz_bug_is_open(1203865):
-            self.skipTest('BZ 1203865: owner_type ignored when creating host')
-        host = entities.Host()
-        host.create_missing()
-        host.owner_type = owner_type
-        host = host.create(create_missing=False)
-        self.assertEqual(host.owner_type, owner_type)
-
-    @data('User', 'Usergroup')
-    def test_update_owner_type(self, owner_type):
-        """@Test: Update a host and specify an ``owner_type``.
+        for owner_type in ('User', 'Usergroup'):
+            with self.subTest(owner_type):
+                if owner_type == 'Usergroup' and bz_bug_is_open(1203865):
+                    continue  # instead of skip for compatibility with nose
+                host = entities.Host()
+                host.create_missing()
+                host.owner_type = owner_type
+                host = host.create(create_missing=False)
```

Comment:
Typically, `create_missing` only populates required attributes that have not yet been filled in. The one exception to this is `Host.create_missing`. It will only work if absolutely no attributes have been filled in. This is because creating missing attributes for a host is extremely complex, much more so than for any other entity. Our solution is to call `create_missing` first an *then* make some customizations.

The update test has an entirely different structure. It goes like this:

```
create a host  # create_missing called here
for owner_type in owner_types:
    update host's owner_type
```

So there's no point to work around the limitations of `Host.create_missing` in that test.